src/jukebox/components/rpc_command_alias.py

```diff
@@ -57,6 +63,24 @@
         'plugin': 'ctrl',
         'method': 'toggle',
         'ignore_card_removal_action': True},
+    'toggle_shuffle': {
+        'package': 'player',
+        'plugin': 'ctrl',
+        'method': 'toggle_shuffle',
+        'note': 'Toggles Shuffle',
+        'ignore_card_removal_action': True},
+    'toggle_repeat': {
```

Comment:
Do we may also need the `toggle_repeatmode` command with the modes `repeat`, `single`, `deactivate`.
This way the user can bind the de-/activation to different cards, which could sometimes be easier to control.

And maybe then join all option in one command?
`repeatmode` -> `toggle` (like in the player UI), `repeat`, `toggle_repeat`, `single`, `toggle_single`, `deactivate` ?